Sources/ApolloCodegenLib/Templates/OperationDefinitionTemplate.swift

```diff
@@ -55,13 +58,15 @@ struct OperationDefinitionTemplate: OperationTemplateRenderer {
       _ operation: CompilationResult.OperationDefinition,
       identifier: @autoclosure () -> String,
       fragments: OrderedSet<IR.NamedFragment>,
-      config: ApolloCodegen.ConfigurationContext
+      config: ApolloCodegen.ConfigurationContext,
+      accessControlRenderer: @autoclosure () -> String
```

Comment:
This is a static function and therefore cannot get access to any of the state from the caller so the function to generate the access control modifier has to be passed in as a closure so it can be called when it's needed.